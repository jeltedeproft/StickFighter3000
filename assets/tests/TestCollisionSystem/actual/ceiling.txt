(25.0,11.05)
(25.0,12.150001)
(25.0,13.3)
(25.0,14.5)
(25.0,15.75)
(25.0,17.05)
(25.0,18.4)
(25.0,19.8)
(25.0,21.25)
(25.0,22.75)
(25.0,24.3)
(25.0,25.9)
(25.0,27.55)
(25.0,29.25)
(25.0,31.0)
(25.0,32.8)
(25.0,34.649998)
(25.0,36.55)
(25.0,38.5)
(25.0,40.5)
(25.0,42.55)
(25.0,44.649998)
(25.0,46.8)
(25.0,49.0)
(25.0,51.25)
(25.0,53.55)
(25.0,55.899998)
(25.0,58.3)
(25.0,60.75)
(25.0,63.25)
(25.0,65.8)
(25.0,68.4)
(25.0,71.05)
(25.0,73.75)
(25.0,76.5)
(25.0,79.3)
(25.0,82.15)
(25.0,85.05)
(25.0,88.0)
(25.0,91.0)
(25.0,94.05)
(25.0,97.15)
(25.0,100.3)
(25.0,103.5)
(25.0,106.75)
(25.0,110.05)
(25.0,113.4)
(25.0,116.8)
(25.0,120.25)
(25.0,123.75)
(25.0,127.3)
(25.0,130.90001)
(25.0,134.55)
(25.0,138.25)
(25.0,142.0)
(25.0,145.8)
(25.0,149.65001)
(25.0,153.55)
(25.0,157.5)
(25.0,161.5)
(25.0,165.55)
(25.0,169.65001)
(25.0,173.8)
(25.0,178.0)
(25.0,182.25)
(25.0,186.55)
(25.0,190.90001)
(25.0,195.3)
(25.0,199.75)
(25.0,204.25)
(25.0,208.8)
(25.0,213.40001)
(25.0,218.05)
(25.0,222.75)
(25.0,227.5)
(25.0,232.3)
(25.0,237.15001)
(25.0,242.05)
(25.0,247.0)
(25.0,252.0)
(25.0,257.05)
(25.0,262.15)
(25.0,267.3)
(25.0,272.5)
(25.0,277.75)
(25.0,283.05)
(25.0,288.4)
(25.0,293.8)
(25.0,299.25)
(25.0,304.75)
(25.0,310.3)
(25.0,315.9)
(25.0,321.55)
(25.0,327.25)
(25.0,333.0)
(25.0,338.8)
(25.0,344.65)
(25.0,350.55)
(25.0,356.5)
(25.0,362.5)
(25.0,368.55)
(25.0,374.65)
(25.0,380.8)
(25.0,387.0)
(25.0,393.25)
(25.0,399.55)
(25.0,405.9)
(25.0,412.3)
(25.0,418.75)
(25.0,425.25)
(25.0,431.8)
(25.0,438.4)
(25.0,445.05)
(25.0,451.75)
(25.0,458.5)
(25.0,465.3)
(25.0,472.15)
(25.0,479.05)
(25.0,486.0)
(25.0,493.0)
(25.0,500.05)
(25.0,507.15)
(25.0,514.3)
(25.0,521.5)
(25.0,528.75)
(25.0,536.05)
(25.0,543.39996)
(25.0,550.8)
(25.0,558.25)
(25.0,565.75)
(25.0,573.3)
(25.0,580.89996)
(25.0,588.55)
(25.0,596.25)
(25.0,604.0)
(25.0,611.8)
(25.0,619.64996)
(25.0,627.55)
(25.0,635.5)
(25.0,643.5)
(25.0,651.55)
(25.0,659.64996)
(25.0,667.8)
(25.0,676.0)
(25.0,684.25)
(25.0,692.55)
(25.0,700.89996)
(25.0,709.3)
(25.0,717.75)
(25.0,726.25)
(25.0,734.8)
(25.0,743.39996)
(25.0,752.05)
(25.0,760.75)
(25.0,769.5)
(25.0,778.3)
(25.0,787.14996)
(25.0,796.05)
(25.0,805.0)
(25.0,814.0)
(25.0,823.05)
(25.0,832.14996)
(25.0,841.3)
(25.0,850.5)
(25.0,859.75)
(25.0,869.05)
(25.0,878.39996)
(25.0,887.8)
(25.0,897.25)
(25.0,906.75)
(25.0,916.3)
(25.0,925.89996)
(25.0,935.55)
(25.0,945.25)
(25.0,955.0)
(25.0,964.8)
(25.0,974.64996)
(25.0,984.55)
(25.0,994.5)
(25.0,1004.5)
(25.0,1014.55)
(25.0,1024.65)
(25.0,1034.8)
(25.0,1045.0)
(25.0,1055.25)
(25.0,1065.55)
(25.0,1075.9)
(25.0,1086.3)
(25.0,1096.75)
(25.0,1107.25)
(25.0,1117.8)
(25.0,1128.4)
(25.0,1139.05)
(25.0,1149.75)
(25.0,1160.5)
(25.0,1171.3)
(25.0,1182.15)
(25.0,1193.05)
(25.0,1204.0)
(25.0,1215.0)
(25.0,1226.05)
(25.0,1237.15)
(25.0,1248.3)
(25.0,1259.5)
(25.0,1270.75)
(25.0,1282.05)
(25.0,1293.4)
(25.0,1304.8)
(25.0,1316.25)
(25.0,1327.75)
(25.0,1339.3)
(25.0,1350.9)
(25.0,1362.55)
(25.0,1374.25)
(25.0,1386.0)
(25.0,1397.8)
(25.0,1409.65)
(25.0,1421.55)
(25.0,1433.5)
(25.0,1445.5)
(25.0,1457.55)
(25.0,1469.65)
(25.0,1481.8)
(25.0,1494.0)
(25.0,1506.25)
(25.0,1518.55)
(25.0,1530.9)
(25.0,1543.3)
(25.0,1555.75)
(25.0,1568.25)
(25.0,1580.8)
(25.0,1593.4)
(25.0,1606.05)
(25.0,1618.75)
(25.0,1631.5)
(25.0,1644.3)
(25.0,1657.15)
(25.0,1670.05)
(25.0,1683.0)
(25.0,1696.0)
(25.0,1709.05)
(25.0,1722.15)
(25.0,1735.3)
(25.0,1748.5)
(25.0,1761.75)
(25.0,1775.05)
(25.0,1788.4)
(25.0,1801.8)
(25.0,1815.25)
(25.0,1828.75)
(25.0,1842.3)
(25.0,1855.9)
(25.0,1869.55)
(25.0,1883.25)
(25.0,1897.0)
(25.0,1910.8)
(25.0,1924.65)
(25.0,1938.55)
(25.0,1952.5)
(25.0,1966.5)
(25.0,1980.55)
(25.0,1994.65)
(25.0,2008.8)
(25.0,2023.0)
(25.0,2037.25)
(25.0,2051.55)
(25.0,2065.9001)
(25.0,2080.3)
(25.0,2094.75)
(25.0,2109.25)
(25.0,2123.8)
(25.0,2138.4001)
(25.0,2153.05)
(25.0,2167.75)
(25.0,2182.5)
(25.0,2197.3)
(25.0,2212.1501)
(25.0,2227.05)
(25.0,2242.0)
(25.0,2257.0)
(25.0,2272.05)
(25.0,2287.1501)
(25.0,2302.3)
(25.0,2317.5)
(25.0,2332.75)
(25.0,2348.05)
(25.0,2363.4001)
(25.0,2378.8)
(25.0,2394.25)
(25.0,2409.75)
(25.0,2425.3)
(25.0,2440.9001)
(25.0,2456.55)
(25.0,2472.25)
(25.0,2488.0)
(25.0,2503.8)
(25.0,2519.6501)
(25.0,2535.55)
(25.0,2551.5)
(25.0,2567.5)
(25.0,2583.55)
(25.0,2599.6501)
(25.0,2615.8)
(25.0,2632.0)
(25.0,2648.25)
(25.0,2664.55)
(25.0,2680.9001)
(25.0,2697.3)
(25.0,2713.75)
(25.0,2730.25)
(25.0,2746.8)
(25.0,2763.4001)
(25.0,2780.05)
(25.0,2796.75)
(25.0,2813.5)
(25.0,2830.3)
(25.0,2847.1501)
(25.0,2864.05)
(25.0,2881.0)
(25.0,2898.0)
(25.0,2915.05)
(25.0,2932.1501)
(25.0,2949.3)
(25.0,2966.5)
(25.0,2983.75)
(25.0,3001.05)
(25.0,3018.4001)
(25.0,3035.8)
(25.0,3053.25)
(25.0,3070.75)
(25.0,3088.3)
(25.0,3105.9001)
(25.0,3123.55)
(25.0,3141.25)
(25.0,3159.0)
(25.0,3176.8)
(25.0,3194.6501)
(25.0,3212.55)
(25.0,3230.5)
(25.0,3248.5)
(25.0,3266.55)
(25.0,3284.6501)
(25.0,3302.8)
(25.0,3321.0)
(25.0,3339.25)
(25.0,3357.55)
(25.0,3375.9001)
(25.0,3394.3)
(25.0,3412.75)
(25.0,3431.25)
(25.0,3449.8)
(25.0,3468.4001)
(25.0,3487.05)
(25.0,3505.75)
(25.0,3524.5)
(25.0,3543.3)
(25.0,3562.1501)
(25.0,3581.05)
(25.0,3600.0)
(25.0,3619.0)
(25.0,3638.05)
(25.0,3657.1501)
(25.0,3676.3)
(25.0,3695.5)
(25.0,3714.75)
(25.0,3734.05)
(25.0,3753.4001)
(25.0,3772.8)
(25.0,3792.25)
(25.0,3811.75)
(25.0,3831.3)
(25.0,3850.9001)
(25.0,3870.55)
(25.0,3890.25)
(25.0,3910.0)
(25.0,3929.8)
(25.0,3949.6501)
(25.0,3969.55)
(25.0,3989.5)
(25.0,4009.5)
(25.0,4029.55)
(25.0,4049.6501)
(25.0,4069.8)
(25.0,4090.0)
(25.0,4110.25)
(25.0,4130.55)
(25.0,4150.9)
(25.0,4171.3)
(25.0,4191.75)
(25.0,4212.25)
(25.0,4232.8)
(25.0,4253.4)
(25.0,4274.05)
(25.0,4294.75)
(25.0,4315.5)
(25.0,4336.3)
(25.0,4357.15)
(25.0,4378.05)
(25.0,4399.0)
(25.0,4420.0)
(25.0,4441.05)
(25.0,4462.15)
(25.0,4483.3)
(25.0,4504.5)
(25.0,4525.75)
(25.0,4547.05)
(25.0,4568.4)
(25.0,4589.8)
(25.0,4611.25)
(25.0,4632.75)
(25.0,4654.3)
(25.0,4675.9)
(25.0,4697.55)
(25.0,4719.25)
(25.0,4741.0)
(25.0,4762.8)
(25.0,4784.65)
(25.0,4806.55)
(25.0,4828.5)
(25.0,4850.5)
(25.0,4872.55)
(25.0,4894.65)
(25.0,4916.8)
(25.0,4939.0)
(25.0,4961.25)
(25.0,4983.55)
(25.0,5005.9)
(25.0,5028.3)
(25.0,5050.75)
(25.0,5073.25)
(25.0,5095.8)
(25.0,5118.4)
(25.0,5141.05)
(25.0,5163.75)
(25.0,5186.5)
(25.0,5209.3)
(25.0,5232.15)
(25.0,5255.05)
(25.0,5278.0)
(25.0,5301.0)
(25.0,5324.05)
(25.0,5347.15)
(25.0,5370.3)
(25.0,5393.5)
(25.0,5416.75)
(25.0,5440.05)
(25.0,5463.4)
(25.0,5486.8)
(25.0,5510.25)
(25.0,5533.75)
(25.0,5557.3)
(25.0,5580.9)
(25.0,5604.55)
(25.0,5628.25)
(25.0,5652.0)
(25.0,5675.8)
(25.0,5699.65)
(25.0,5723.55)
(25.0,5747.5)
(25.0,5771.5)
(25.0,5795.55)
(25.0,5819.65)
(25.0,5843.8)
(25.0,5868.0)
(25.0,5892.25)
(25.0,5916.55)
(25.0,5940.9)
(25.0,5965.3)
(25.0,5989.75)
(25.0,6014.25)
(25.0,6038.8)
(25.0,6063.4)
(25.0,6088.05)
(25.0,6112.75)
(25.0,6137.5)
(25.0,6162.3)
(25.0,6187.15)
(25.0,6212.05)
(25.0,6237.0)
(25.0,6262.0)
(25.0,6287.05)
(25.0,6312.15)
(25.0,6337.3)
(25.0,6362.5)
(25.0,6387.75)
(25.0,6413.05)
(25.0,6438.4)
(25.0,6463.8)
(25.0,6489.25)
(25.0,6514.75)
(25.0,6540.3)
(25.0,6565.9)
(25.0,6591.55)
(25.0,6617.25)
(25.0,6643.0)
(25.0,6668.8)
(25.0,6694.65)
(25.0,6720.55)
(25.0,6746.5)
(25.0,6772.5)
(25.0,6798.55)
(25.0,6824.65)
(25.0,6850.8)
(25.0,6877.0)
(25.0,6903.25)
(25.0,6929.55)
(25.0,6955.9)
(25.0,6982.3)
(25.0,7008.75)
(25.0,7035.25)
(25.0,7061.8)
(25.0,7088.4)
(25.0,7115.05)
(25.0,7141.75)
(25.0,7168.5)
(25.0,7195.3)
(25.0,7222.15)
(25.0,7249.05)
(25.0,7276.0)
(25.0,7303.0)
(25.0,7330.05)
(25.0,7357.15)
(25.0,7384.3)
(25.0,7411.5)
(25.0,7438.75)
(25.0,7466.05)
(25.0,7493.4)
(25.0,7520.8)
(25.0,7548.25)
(25.0,7575.75)
(25.0,7603.3)
(25.0,7630.9)
(25.0,7658.55)
(25.0,7686.25)
(25.0,7714.0)
(25.0,7741.8)
(25.0,7769.65)
(25.0,7797.55)
(25.0,7825.5)
(25.0,7853.5)
(25.0,7881.55)
(25.0,7909.65)
(25.0,7937.8)
(25.0,7966.0)
(25.0,7994.25)
(25.0,8022.55)
(25.0,8050.9)
(25.0,8079.3)
(25.0,8107.75)
(25.0,8136.25)
(25.0,8164.8)
(25.0,8193.399)
(25.0,8222.05)
(25.0,8250.75)
(25.0,8279.5)
(25.0,8308.3)
(25.0,8337.149)
(25.0,8366.05)
(25.0,8395.0)
(25.0,8424.0)
(25.0,8453.05)
(25.0,8482.149)
(25.0,8511.3)
(25.0,8540.5)
(25.0,8569.75)
(25.0,8599.05)
(25.0,8628.399)
(25.0,8657.8)
(25.0,8687.25)
(25.0,8716.75)
(25.0,8746.3)
(25.0,8775.899)
(25.0,8805.55)
(25.0,8835.25)
(25.0,8865.0)
(25.0,8894.8)
(25.0,8924.649)
(25.0,8954.55)
(25.0,8984.5)
(25.0,9014.5)
(25.0,9044.55)
(25.0,9074.649)
(25.0,9104.8)
(25.0,9135.0)
(25.0,9165.25)
(25.0,9195.55)
(25.0,9225.899)
(25.0,9256.3)
(25.0,9286.75)
(25.0,9317.25)
(25.0,9347.8)
(25.0,9378.399)
(25.0,9409.05)
(25.0,9439.75)
(25.0,9470.5)
(25.0,9501.3)
(25.0,9532.149)
(25.0,9563.05)
(25.0,9594.0)
(25.0,9625.0)
(25.0,9656.05)
(25.0,9687.149)
(25.0,9718.3)
(25.0,9749.5)
(25.0,9780.75)
(25.0,9812.05)
(25.0,9843.399)
(25.0,9874.8)
(25.0,9906.25)
(25.0,9937.75)
(25.0,9969.3)
(25.0,10000.899)
(25.0,10032.55)
(25.0,10064.25)
(25.0,10096.0)
(25.0,10127.8)
(25.0,10159.649)
(25.0,10191.55)
(25.0,10223.5)
(25.0,10255.5)
(25.0,10287.55)
(25.0,10319.649)
(25.0,10351.8)
(25.0,10384.0)
(25.0,10416.25)
(25.0,10448.55)
(25.0,10480.899)
(25.0,10513.3)
(25.0,10545.75)
(25.0,10578.25)
(25.0,10610.8)
(25.0,10643.399)
(25.0,10676.05)
(25.0,10708.75)
(25.0,10741.5)
(25.0,10774.3)
(25.0,10807.149)
(25.0,10840.05)
(25.0,10873.0)
(25.0,10906.0)
(25.0,10939.05)
(25.0,10972.149)
(25.0,11005.3)
(25.0,11038.5)
(25.0,11071.75)
(25.0,11105.05)
(25.0,11138.399)
(25.0,11171.8)
(25.0,11205.25)
(25.0,11238.75)
(25.0,11272.3)
(25.0,11305.899)
(25.0,11339.55)
(25.0,11373.25)
(25.0,11407.0)
(25.0,11440.8)
(25.0,11474.649)
(25.0,11508.55)
(25.0,11542.5)
(25.0,11576.5)
(25.0,11610.55)
(25.0,11644.649)
(25.0,11678.8)
(25.0,11713.0)
(25.0,11747.25)
(25.0,11781.55)
(25.0,11815.899)
(25.0,11850.3)
(25.0,11884.75)
(25.0,11919.25)
(25.0,11953.8)
(25.0,11988.399)
(25.0,12023.05)
(25.0,12057.75)
(25.0,12092.5)
(25.0,12127.3)
(25.0,12162.149)
(25.0,12197.05)
(25.0,12232.0)
(25.0,12267.0)
(25.0,12302.05)
(25.0,12337.149)
(25.0,12372.3)
(25.0,12407.5)
(25.0,12442.75)
(25.0,12478.05)
(25.0,12513.399)
(25.0,12548.8)
(25.0,12584.25)
(25.0,12619.75)
(25.0,12655.3)
(25.0,12690.899)
(25.0,12726.55)
(25.0,12762.25)
(25.0,12798.0)
(25.0,12833.8)
(25.0,12869.649)
(25.0,12905.55)
(25.0,12941.5)
(25.0,12977.5)
(25.0,13013.55)
(25.0,13049.649)
(25.0,13085.8)
(25.0,13122.0)
(25.0,13158.25)
(25.0,13194.55)
(25.0,13230.899)
(25.0,13267.3)
(25.0,13303.75)
(25.0,13340.25)
(25.0,13376.8)
(25.0,13413.399)
(25.0,13450.05)
(25.0,13486.75)
(25.0,13523.5)
(25.0,13560.3)
(25.0,13597.149)
(25.0,13634.05)
(25.0,13671.0)
(25.0,13708.0)
(25.0,13745.05)
(25.0,13782.149)
(25.0,13819.3)
(25.0,13856.5)
(25.0,13893.75)
(25.0,13931.05)
(25.0,13968.399)
(25.0,14005.8)
(25.0,14043.25)
(25.0,14080.75)
(25.0,14118.3)
(25.0,14155.899)
(25.0,14193.55)
(25.0,14231.25)
(25.0,14269.0)
(25.0,14306.8)
(25.0,14344.649)
(25.0,14382.55)
(25.0,14420.5)
(25.0,14458.5)
(25.0,14496.55)
(25.0,14534.649)
(25.0,14572.8)
(25.0,14611.0)
(25.0,14649.25)
(25.0,14687.55)
(25.0,14725.899)
(25.0,14764.3)
(25.0,14802.75)
(25.0,14841.25)
(25.0,14879.8)
(25.0,14918.399)
(25.0,14957.05)
(25.0,14995.75)
(25.0,15034.5)
(25.0,15073.3)
(25.0,15112.149)
(25.0,15151.05)
(25.0,15190.0)
(25.0,15229.0)
(25.0,15268.05)
(25.0,15307.149)
(25.0,15346.3)
(25.0,15385.5)
(25.0,15424.75)
(25.0,15464.05)
(25.0,15503.399)
(25.0,15542.8)
(25.0,15582.25)
(25.0,15621.75)
(25.0,15661.3)
(25.0,15700.899)
(25.0,15740.55)
(25.0,15780.25)
(25.0,15820.0)
(25.0,15859.8)
(25.0,15899.649)
(25.0,15939.55)
(25.0,15979.5)
(25.0,16019.5)
(25.0,16059.55)
(25.0,16099.649)
(25.0,16139.8)
(25.0,16180.0)
(25.0,16220.25)
(25.0,16260.55)
(25.0,16300.899)
(25.0,16341.3)
(25.0,16381.75)
(25.0,16422.25)
(25.0,16462.8)
(25.0,16503.4)
(25.0,16544.05)
(25.0,16584.75)
(25.0,16625.5)
(25.0,16666.3)
(25.0,16707.15)
(25.0,16748.05)
(25.0,16789.0)
(25.0,16830.0)
(25.0,16871.05)
(25.0,16912.15)
(25.0,16953.3)
(25.0,16994.5)
(25.0,17035.75)
(25.0,17077.05)
(25.0,17118.4)
(25.0,17159.8)
(25.0,17201.25)
(25.0,17242.75)
(25.0,17284.3)
(25.0,17325.9)
(25.0,17367.55)
(25.0,17409.25)
(25.0,17451.0)
(25.0,17492.8)
(25.0,17534.65)
(25.0,17576.55)
(25.0,17618.5)
(25.0,17660.5)
(25.0,17702.55)
(25.0,17744.65)
(25.0,17786.8)
(25.0,17829.0)
(25.0,17871.25)
(25.0,17913.55)
(25.0,17955.9)
(25.0,17998.3)
(25.0,18040.75)
(25.0,18083.25)
(25.0,18125.8)
(25.0,18168.4)
(25.0,18211.05)
(25.0,18253.75)
(25.0,18296.5)
(25.0,18339.3)
(25.0,18382.15)
(25.0,18425.05)
(25.0,18468.0)
(25.0,18511.0)
(25.0,18554.05)
(25.0,18597.15)
(25.0,18640.3)
(25.0,18683.5)
(25.0,18726.75)
(25.0,18770.05)
(25.0,18813.4)
(25.0,18856.8)
(25.0,18900.25)
(25.0,18943.75)
(25.0,18987.3)
(25.0,19030.9)
(25.0,19074.55)
(25.0,19118.25)
(25.0,19162.0)
(25.0,19205.8)
(25.0,19249.65)
(25.0,19293.55)
(25.0,19337.5)
(25.0,19381.5)
(25.0,19425.55)
(25.0,19469.65)
(25.0,19513.8)
(25.0,19558.0)
(25.0,19602.25)
(25.0,19646.55)
(25.0,19690.9)
(25.0,19735.3)
(25.0,19779.75)
(25.0,19824.25)
(25.0,19868.8)
(25.0,19913.4)
(25.0,19958.05)
(25.0,20002.75)
(25.0,20047.5)
(25.0,20092.3)
(25.0,20137.15)
(25.0,20182.05)
(25.0,20227.0)
(25.0,20272.0)
(25.0,20317.05)
(25.0,20362.15)
(25.0,20407.3)
(25.0,20452.5)
(25.0,20497.75)
(25.0,20543.05)
(25.0,20588.4)
(25.0,20633.8)
(25.0,20679.25)
(25.0,20724.75)
(25.0,20770.3)
(25.0,20815.9)
(25.0,20861.55)
(25.0,20907.25)
(25.0,20953.0)
(25.0,20998.8)
(25.0,21044.65)
(25.0,21090.55)
(25.0,21136.5)
(25.0,21182.5)
(25.0,21228.55)
(25.0,21274.65)
(25.0,21320.8)
(25.0,21367.0)
(25.0,21413.25)
(25.0,21459.55)
(25.0,21505.9)
(25.0,21552.3)
(25.0,21598.75)
(25.0,21645.25)
(25.0,21691.8)
(25.0,21738.4)
(25.0,21785.05)
(25.0,21831.75)
(25.0,21878.5)
(25.0,21925.3)
(25.0,21972.15)
(25.0,22019.05)
(25.0,22066.0)
(25.0,22113.0)
(25.0,22160.05)
(25.0,22207.15)
(25.0,22254.3)
(25.0,22301.5)
(25.0,22348.75)
(25.0,22396.05)
(25.0,22443.4)
(25.0,22490.8)
(25.0,22538.25)
(25.0,22585.75)
(25.0,22633.3)
(25.0,22680.9)
(25.0,22728.55)
(25.0,22776.25)
(25.0,22824.0)
(25.0,22871.8)
(25.0,22919.65)
(25.0,22967.55)
(25.0,23015.5)
(25.0,23063.5)
(25.0,23111.55)
(25.0,23159.65)
(25.0,23207.8)
(25.0,23256.0)
(25.0,23304.25)
(25.0,23352.55)
(25.0,23400.9)
(25.0,23449.3)
(25.0,23497.75)
(25.0,23546.25)
(25.0,23594.8)
(25.0,23643.4)
(25.0,23692.05)
(25.0,23740.75)
(25.0,23789.5)
(25.0,23838.3)
(25.0,23887.15)
(25.0,23936.05)
(25.0,23985.0)
(25.0,24034.0)
(25.0,24083.05)
(25.0,24132.15)
(25.0,24181.3)
(25.0,24230.5)
(25.0,24279.75)
(25.0,24329.05)
(25.0,24378.4)
(25.0,24427.8)
(25.0,24477.25)
(25.0,24526.75)
(25.0,24576.3)
(25.0,24625.9)
(25.0,24675.55)
(25.0,24725.25)
(25.0,24775.0)
(25.0,24824.8)
(25.0,24874.65)
(25.0,24924.55)
(25.0,24974.5)
(25.0,25024.5)
(25.0,25074.5)
(25.0,25124.5)
(25.0,25174.5)
(25.0,25224.5)
(25.0,25274.5)
(25.0,25324.5)
(25.0,25374.5)
(25.0,25424.5)
(25.0,25474.5)
(25.0,25524.5)
(25.0,25574.5)
(25.0,25624.5)
(25.0,25674.5)
(25.0,25724.5)
(25.0,25774.5)
(25.0,25824.5)
(25.0,25874.5)
(25.0,25924.5)
(25.0,25974.5)
(25.0,26024.5)
