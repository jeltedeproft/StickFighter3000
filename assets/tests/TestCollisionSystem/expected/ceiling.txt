collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

collidable 1 : STATIC_TOP
collidable 2 : PLAYER

